dynawo/sources/Models/Modelica/Dynawo/Examples/DynaFlow/IEEE14/Grid/BaseClasses/IEEE14Base.mo

```diff
@@ -0,0 +1,261 @@
+
+  // Generators
+  Electrical.Machines.SignalN.GeneratorPV Gen1(KGover = 1, PGen0Pu = 2.3239, PMaxPu = 10.9, PMinPu = 0, PNom = 1090, PRef0Pu = -2.3239, QGen0Pu = -0.1655, QMaxPu = 100, QMinPu = -100, QNomAlt = 10000, U0Pu = 1.06, URef0Pu = 1.06, i0Pu = Complex(-2.192358, -0.156132), limUQDown0 = false, limUQUp0 = false, qStatus0 = Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, u0Pu = Complex(1.06, 0)) annotation(
+    Placement(visible = true, transformation(origin = {-170, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Machines.SignalN.GeneratorPV Gen2(KGover = 1, PGen0Pu = 0.4, PMaxPu = 10.08, PMinPu = 0, PNom = 1008, PRef0Pu = -0.4, QGen0Pu = 0.4356, QMaxPu = 100, QMinPu = -100, QNomAlt = 10000, U0Pu = 1.045072, URef0Pu = 1.045072, i0Pu = Complex(-0.345121, 0.448465), limUQDown0 = false, limUQUp0 = false, qStatus0 = Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, u0Pu = Complex(1.041127, -0.090721)) annotation(
+    Placement(visible = true, transformation(origin = {-90, -160}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Machines.SignalN.GeneratorPV Gen3(KGover = 0, PGen0Pu = 0, PMaxPu = 14.85, PMinPu = 0, PNom = 1485, PRef0Pu = 0, QGen0Pu = 0.250700, QMaxPu = 100, QMinPu = -100, QNomAlt = 10000, U0Pu = 1.01, URef0Pu = 1.01, i0Pu = Complex(0.054697, 0.242116), limUQDown0 = false, limUQUp0 = false, qStatus0 = Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, u0Pu = Complex(0.985173, -0.222561)) annotation(
+    Placement(visible = true, transformation(origin = {90, -200}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Machines.SignalN.GeneratorPV Gen6(KGover = 0, PGen0Pu = 0, PMaxPu = 0.744, PMinPu = 0, PNom = 74.4, PRef0Pu = 0, QGen0Pu = 0.1273, QMaxPu = 100, QMinPu = -100, QNomAlt = 10000, U0Pu = 1.070290, URef0Pu = 1.070290, i0Pu = Complex(0.029217, 0.115295), limUQDown0 = false, limUQUp0 = false, qStatus0 = Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, u0Pu = Complex(1.037496, -0.262912)) annotation(
+    Placement(visible = true, transformation(origin = {-60, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Machines.SignalN.GeneratorPV Gen8(KGover = 0, PGen0Pu = 0, PMaxPu = 2.28, PMinPu = 0, PNom = 228, PRef0Pu = 0, QGen0Pu = 0.1762, QMaxPu = 100, QMinPu = -100, QNomAlt = 10000, U0Pu = 1.089855, URef0Pu = 1.089855, i0Pu = Complex(0.037358, 0.157298), limUQDown0 = false, limUQUp0 = false, qStatus0 = Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, u0Pu = Complex(1.060361, -0.251831)) annotation(
+    Placement(visible = true, transformation(origin = {170, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  // Loads
+  Electrical.Loads.LoadAlphaBetaRestorative Load2(Alpha = 1.5, Beta = 2.5, UMaxPu = 1.05, UMinPu = 0.95, i0Pu = Complex(0.196308, -0.139089), s0Pu = Complex(0.217000, 0.127000), tFilter = 10, u0Pu = Complex(1.041127, -0.090721)) annotation(
```

Comment:
You could add a parameter Alpha and a parameter Beta on the IEEE14Base model, and use it for all your loads, instead of writing 1.5 and 2.5 several times. Same for UMaxPu and UMinPu (with a comment for each parameter of course).

Reply:
And for tFilter too